src/main/java/org/apache/sling/event/impl/jobs/stats/GaugeSupport.java

```diff
@@ -48,6 +50,8 @@ class GaugeSupport {
     private final Set<String> gaugeMetricNames = new HashSet<>();
     private final String queueName;
 
+    Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
This should be private (and probably static final) most Sling projects use log or LOG as name the convention.